test/command-server.test.ts

```diff
@@ -143,31 +143,74 @@ describe('command server', () => {
         assertions: []
       };
 
-      atom.update(assoc('manifest', [
-        { path: "foo.js", test: test1 },
-        { path: "bar.js", test: test2 },
-      ]));
-      result = await query('manifest { path, test }');
+      atom.update(assoc('manifest', {
+        url: 'http://bigmanifest.com',
+        entries: [
+          { path: "foo.js", test: test1 },
+          { path: "bar.js", test: test2 }
+        ],
+      }));
+      result = await query(`
+manifest {
+  url
+  sources
+  suite {
+    id
+    description
+    children {
+      id description
+      children {
+        id description
+      }
+    }
+  }
+}
+`);
     });
+
     it('contains the paths of the tests', () => {
       expect(result).toMatchObject({
         data: {
-          manifest: [
-            { path: "foo.js" },
-            { path: "bar.js" },
-          ]
+          manifest: {
+            sources: [
+              "foo.js",
+              "bar.js"
+            ]
+          }
         }
       })
     });
-    it('contains the JSON encoding of the test tree', () => {
-      let [first, second]: Array<SerializableTest> = result.data.manifest.map(m => JSON.parse(m.test));
 
-      expect(first.description).toEqual('First Test');
-      expect(first.steps).toEqual([ { description: "Do the thing" }]);
-      expect(first.children).toMatchObject([ { description: "Son of First Test" }]);
-      expect(first.assertions).toMatchObject([ { description: "It did the thing" }]);
+    it('contains the test tree', () => {
+      expect(result).toMatchObject({
+        data: {
+          manifest: {
+            suite: {
+              id: "",
+              description: "All Tests",
+              children: [{
+                id: "First Test",
+                description: "First Test",
+                children: [{
+                  id: "First Test > Son of First Test",
+                  description: "Son of First Test"
+                }]
+              }, {
+                id: "Second Test",
+                description: "Second Test"
+              }]
+            }
+          }
+        }
+      })
+      // let [first, second]: Array<> = result.data.manifest.map(m => JSON.parse(m.test));
+
+      // expect(first.description).toEqual('First Test');
+      // expect(first.steps).toEqual([ { description: "Do the thing" }]);
+      // expect(first.children).toMatchObject([ { description: "Son of First Test" }]);
+      // expect(first.assertions).toMatchObject([ { description: "It did the thing" }]);
```

Comment:
Are these commented assertions supposed to be here?

Reply:
Me ->🤦‍♂